src/components/CommentTeaser/CommentTeaser.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { css, merge } from 'glamor'
+import colors from '../../theme/colors'
+import { serifRegular14 } from '../Typography/styles'
+import CommentHeader from '../Comment/CommentHeader'
+import CommentTeaserHeader from './CommentTeaserHeader'
+import CommentTeaserFooter from './CommentTeaserFooter'
+
+const styles = {
+  root: {
+    marginBottom: '10px',
+    '& + &': {
+      borderTop: `1px solid ${colors.divider}`,
+      paddingTop: '20px'
+    }
+  },
+  body: {
+    ...serifRegular14,
+    margin: '10px 0'
+  },
+  clamp: {
+    // TODO: Replace with a cross-browser JS solution for line-clamping.
+    overflow: 'hidden',
+    display: '-webkit-box',
+    WebkitBoxOrient: 'vertical'
+  },
+  box: {
+    border: `1px solid ${colors.divider}`,
+    padding: '20px'
+  }
+}
+
+export const CommentTeaser = ({
+  t,
+  title,
+  subtitle,
+  displayAuthor,
+  content,
+  timeago,
+  commentUrl,
+  lineClamp,
+  isBox
+}) => (
+  <div {...css(merge(styles.root, isBox ? styles.box : {}))}>
```

Comment:
I noticed that you call `css()` here in the component render function instead of when constructing the `styles` object (eg. https://github.com/orbiting/styleguide/blob/e9ffc111659165b619e364d26e05050f5589ada6/src/components/Comment/CommentHeader.js#L7-L8). Is there any advantage / disadvantage?

Reply:
Actually, I've now removed css() completely here. "merge" just seems to be a semantic alias for "css" anyway, so it doesn't make sense to use both. The reason why I'm doing it in the render function is because I want to modify styles at render time depending on properties like "isBox".